code/game/machinery/computer/orders/order_computer/mining_order.dm

```diff
@@ -84,6 +76,22 @@
 	if(!.)
 		flick("mining-deny", src)
 
+/obj/machinery/computer/order_console/mining/ui_static_data(mob/user)
+	var/list/data = ..()
+	data["order_datums"] = list()
+	for(var/datum/orderable_item/item as anything in GLOB.order_console_products)
+		if(!(item.category_index in order_categories))
+			continue
+		data["order_datums"] += list(list(
+			"name" = item.name,
+			"desc" = item.desc,
+			"cat" = item.category_index,
+			"ref" = REF(item),
+			"cost" = GET_MINING_SHIPPING_MULTIPLIER(item.cost_per_order),
```

Comment:
This is what I was scared of when you wanted to use defines instead of editing the costs directly, we're filling ``orderable_datums`` twice every static data now, I wanted to keep it as close to the base as I could

Reply:
dont fill twice oki